### PR TITLE
Make fuzzing work.

### LINKED ISSUE #6591: Can not run fuzzer
### What version of TiKV are you using?

master

### What operating system and CPU are you using?

x86_x64

### Steps to reproduce

1. run `cargo run --package fuzz -- run Afl fuzz_codec_bytes`
2. see
```
12T08:58:49.336Z] error[E0599]: no method named `generate_files` found for type `&Builder` in the current scope
[2020-02-12T08:58:49.336Z]   --> /rust/git/checkouts/protobuf-build-174b08b051947a10/42e52b9/src/lib.rs:56:14
[2020-02-12T08:58:49.336Z]    |
[2020-02-12T08:58:49.336Z] 56 |         self.generate_files();
[2020-02-12T08:58:49.336Z]    |              ^^^^^^^^^^^^^^ method not found in `&Builder`
[2020-02-12T08:58:49.336Z] 
[2020-02-12T08:58:49.336Z] error: aborting due to previous error
[2020-02-12T08:58:49.336Z] 
[2020-02-12T08:58:49.336Z] For more information about this error, try `rustc --explain E0599`.
[2020-02-12T08:58:49.336Z] error: could not compile `protobuf-build`.
[2020-02-12T08:58:49.336Z] warning: build failed, waiting for other jobs to finish...
[2020-02-12T08:59:16.192Z] error: build failed
[2020-02-12T08:59:16.192Z] Running fuzzer failed: error building afl instrumented binary, exit code Some(101)
[2020-02-12T08:59:16.192Z] Error: 1
```

### What did you expect?
No error and start fuzz

### PR DESCRIPTION
**Pull request by nrc**
Fixes a Cargo error and adds a new make target, fuzz.

###  What have you changed?

There is a bit of complexity passing the right features to dependencies when fuzzing. This PR fixes that by adding dependencies to fuzz-targets. I also add a make target to make fuzzing more convenient.

###  What is the type of the changes?

- Bugfix (a change which fixes an issue)
- Engineering (engineering change which doesn't change any feature or fix any issue)

###  How is the PR tested?

- Manual test (add detailed scripts or steps below)

`make test` and `FUZZ_TARGET=fuzz_codec_bytes make test`

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
no

###  Does this PR affect `tidb-ansible`?
no

###  Refer to a related PR or issue link (optional)

Fixes #6591

PTAL @zhouqiang-cl @breeswish

### CODE REVIEW

**Comment by breeswish** [comment]
LGTM. However CI failed :(

**Comment by zhouqiang-cl** [comment]
The afl fuzz can work now!!!

**Comment by nrc** [comment]
@breeswish @zhouqiang-cl PTAL. Passing CI now, but the Makefile got a bit more complicated

File: Makefile [hunk 1]
```diff
@@ -226,7 +226,9 @@ test:
 	export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${LOCAL_DIR}/lib" && \
 	export LOG_LEVEL=DEBUG && \
 	export RUST_BACKTRACE=1 && \
-	cargo test --no-default-features --features "${ENABLE_FEATURES}" --all --exclude tests --exclude cdc ${EXTRA_CARGO_ARGS} -- --nocapture && \
+	cargo test --no-default-features --features "${ENABLE_FEATURES}" --all --exclude tests --exclude \
```
Comment (breeswish) [inline Makefile]: why is cdc also excluded?

Comment (nrc) [reply Makefile]: No idea. It was excluded before so I left it excluded

**Comment by zhouqiang-cl** [comment]
@overvenus @breeswish PTAL again

File: Makefile (hunk 1 above)
Comment (overvenus) [reply Makefile]: Because cdc and tests are not included in features `prost/protobuf-codec` in src/Cargo.toml.

**Pull request merged**